sdk/nodejs/examples/pastebin/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dependencies": {
+    "@openziti/ziti-sdk-nodejs": "^0.14.2",
```

Comment:
Doesn't this dependency get picked up automatically?

Reply:
It will, once I add the CI for the `zrok-sdk-nodejs` package (which will embed the `ziti-sdk-nodejs`)